update-compose.yaml

```diff
@@ -2,7 +2,7 @@
 # https://www.updatecli.io/docs/core/compose/
 policies:
   - name: Handle default oblt-cli version
-    policy: ghcr.io/elastic/oblt-updatecli-policies/oblt-cli/version:0.0.2@sha256:dd79537aa64a28c57da351182a70a2a798e9b6c5eb55c243fdbd4f5bffe5133d
+    policy: ghcr.io/elastic/oblt-updatecli-policies/oblt-cli/version:0.0.3@sha256:8b996fc5e4351610117ad87e0cc74f562c8228c967a5dde0ed2f9c28f1935c3c
```

Comment:
Failed, unfortunately. https://github.com/elastic/oblt-actions/actions/runs/9859667131

At least, I can try to fix it in this PR

Reply:
were you able to make the autodiscovery update work in another repo?

I'm wondering if it's failing because it cannot read the oblt-cli/version policy package

Reply:
It worked in https://github.com/elastic/apm-agent-go/pull/1634 when running https://github.com/elastic/apm-agent-go/actions/runs/9809715657 as part of https://github.com/elastic/apm-agent-go/pull/1633, for such I created a feature branch in the upstream

Reply:
I know what's the error about:

> ⚠ While nothing change in the current pipeline run, according to the git history, some commits must be pushed
ERROR: non-fast-forward update
ERROR: push error: unpack error: index-pack failed

There is a missing piece of configuration:
- https://github.com/elastic/apm-agent-go/pull/1633/files#diff-86d859b77504f921867f9a096e88d0886b8ae9df72541cff1f725c420f5e3e1fR6-R9

Reply:
Thank you!

Reply:
https://github.com/elastic/oblt-actions/pull/85/commits/b4f3fa8d6e71232e7b7e542ef4e130bcad98b9da

Reply:
Do you think you can remove the bump and keep the rest of the files so we can test the updatecli itself by running the main branch?

Reply:
https://github.com/elastic/oblt-actions/pull/85/commits/5a899feafb5e976ffb2e57ef1a1380e8279b5fb3